src/networks.json

```diff
@@ -211,18 +211,34 @@
   },
   "50": {
     "key": "50",
-    "name": "XinFin MainNet",
+    "name": "XDC.Network",
     "shortName": "XDC",
     "chainId": 50,
     "network": "mainnet",
-    "multicall": "",
+    "multicall": "0xc8160deb19559d93eadb82798ededce696ebcaf5",
+    "rpc": [
+      "https://snapshotrpc.blocksscan.io/"
```

Comment:
It works the same as `https://rpc.xinfin.network` can we use it? 🤔 (as it is suggested here https://howto.xinfin.org/general/networks/ )